File: WebAppVen/Scripts/app/model/AB0053_2.js
Ext.define('WEBAPP.model.AB0053_2', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MMCODE', type: 'string' }, //院內碼(畫面)
        { name: 'MMNAME_C', type: 'string' }, //中文品名(畫面)
        { name: 'MMNAME_E', type: 'string' }, //英文品名(畫面)
        { name: 'EXP_DATE1', type: 'string' }, //最近效期(畫面)
        { name: 'LOT_NO', type: 'string' }, //藥品批號(畫面)
        { name: 'REPLY_DATE', type: 'string' }, //回覆效期(畫面)
        { name: 'PH1S', type: 'string' }, //藥庫(畫面)
        { name: 'CHEMO', type: 'string' }, //內湖化療調配室(畫面)
        { name: 'CHEMOT', type: 'string' }, //汀洲化療調配室(畫面)
        { name: 'PH1A', type: 'string' }, //內湖住院藥局(畫面)
        { name: 'PH1C', type: 'string' }, //內湖門診藥局(畫面)
        { name: 'PH1R', type: 'string' }, //內湖急診藥局(畫面)
        { name: 'PHMC', type: 'string' }, //汀洲藥局(畫面)
        { name: 'TPN', type: 'string' }, //製劑室(畫面)
        { name: 'BASE_UNIT', type: 'string' }, //劑量單位(畫面)
        { name: 'E_MANUFACT', type: 'string' }, //廠商(畫面)
    ]
});

File: WebApp/Models/AB/AB0053_2.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class AB0053_2 : JCLib.Mvc.BaseModel
    {
        public string MMCODE { get; set; } //院內碼(畫面)
        public string MMNAME_C { get; set; } //中文品名(畫面)
        public string MMNAME_E { get; set; } //英文品名(畫面)
        public string EXP_DATE { get; set; } //最近效期(畫面)
        public string LOT_NO { get; set; } //藥品批號(畫面)
        public string REPLY_DATE { get; set; } //回覆效期(畫面)
        public string PH1S { get; set; } //藥庫(畫面)
        public string CHEMO { get; set; } //內湖化療調配室(畫面)
        public string CHEMOT { get; set; } //汀洲化療調配室(畫面)
        public string PH1A { get; set; } //內湖住院藥局(畫面)
        public string PH1C { get; set; } //內湖門診藥局(畫面)
        public string PH1R { get; set; } //內湖急診藥局(畫面)
        public string PHMC { get; set; } //汀洲藥局(畫面)
        public string TPN { get; set; } //製劑室(畫面)
        public string BASE_UNIT { get; set; } //劑量單位(畫面)
        public string E_MANUFACT { get; set; } //廠商(畫面)
        public string EXP_QTY { get; set; } //總數(畫面)
    }
}
